plugins/repository-s3/build.gradle

```diff
@@ -172,15 +172,13 @@ if (useFixture) {
   // and pass its name instead.
   task writeMinioAddress {
     dependsOn tasks.bundlePlugin, tasks.postProcessFixture
-    outputs.file(minioAddressFile)
```

Comment:
Adding these outputs and inputs was a mistake, repeated runs against minio fail because of them since we need to update the file when repeatedly starting Minio.